std_msgs/Header header
custom_messages/DetectedObject[] detected_object